client/my-sites/media-library/content.jsx

```diff
@@ -182,7 +226,11 @@ const MediaLibraryContent = React.createClass( {
 
 	renderMediaList: function() {
 		if ( ! this.props.site ) {
-			return <MediaLibraryList key="list-loading" />;
+			return <MediaLibraryList key="list-loading" filterRequiresUpgrade={ this.props.filterRequiresUpgrade } />;
```

Comment:
I couldn't test this because I couldn't imagine a situation where `site` is null or undefined: both [media-modal](https://github.com/Automattic/wp-calypso/blob/master/client/post-editor/media-modal/index.jsx#L525) and the library [itself](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/media/main.jsx#L279) check for site being present. But I understand you added the `filterRequiresUpgrade` just to respect the plans own features, as in the other uses of `MediaLibraryList`. It drew my attention because is unrelated to the feature in this PR and I just wanted to make sure this is an intended change.